src/components/CookieConsentBanner/index.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import CookieConsent from 'react-cookie-consent'
+import ModalContainer from '../ModalContainer'
+import ButtonContainer from '../ButtonContainer'
+import PrivacyPolicy from '../PrivacyPolicy'
+import './styles.scss'
+
+const CookieConsentBanner = props => {
+  const { modalIsOpen, closeModal } = props
```

Comment:
User can accidentally close the modal with outside click. Is this intentional or should we disable it? I'm just thinking the scenario where user accidentally closes the modal and does not know how to show it again but can't log in without accepting the cookies.